tests/remote_resource/test_requests.py

```diff
@@ -0,0 +1,257 @@
+import mock
+import json
+from django.test import TestCase
+from zc_common.remote_resource.request import \
+    RemoteResourceWrapper, RemoteResourceListWrapper, get_route_from_fk, make_service_request, get_remote_resource
+
+
+class WrappersTestCase(TestCase):
+    def test_remote_resource_wrapper_one_relationship(self):
+        data = {
+            "data": {
+                "type": "articles",
+                "id": "1",
+                "attributes": {
+                    "title": "Omakase"
+                },
+                "relationships": {
+                    "author": {
+                        "links": {
+                            "self": "/articles/1/relationships/author",
+                            "related": "/articles/1/author"
+                        },
+                        "data": {"type": "People", "id": "9"}
+                    }
+                }
+            }
+        }
+
+        resource = RemoteResourceWrapper(data['data'])
+
+        self.assertEqual(resource.title, "Omakase")
+        self.assertEqual(resource.author.type, "People")
+        self.assertEqual(resource.author.id, "9")
```

Comment:
For these, I'd probably add 2 assertions that `resource.type` is `articles` and `resource.id` is `1` as well, just to make sure those values are getting set properly.